sdk/sdk-view/src/test/java/mil/emp3/core/StorageManagerTest.java

```diff
@@ -0,0 +1,462 @@
+package mil.emp3.core;
```

Comment:
There are many interpretations of Unit Tests. So far all the unit tests have been added to exercise the EMP code from API level, which provides the maximum code coverage. Looks like we are departing from that model to test each of the classes. Why not test the userContext capability starting at the API?